build/drone/ubuntu_oss.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+set -ue
+
+rm -rf non-oss/*
+
+# build binary
+apt-get update -y
+apt-get install -y -q squashfs-tools build-essential ruby bison ruby-dev git-core texinfo curl
+curl -sL https://dl.bintray.com/kontena/ruby-packer/0.5.0-dev/rubyc-linux-amd64.gz | gunzip > /usr/local/bin/rubyc
+chmod +x /usr/local/bin/rubyc
+gem install bundler
+version=${DRONE_TAG#"v"}
+package="pharos-cluster-oss-linux-amd64-${version}"
+rubyc -o $package -d /__enclose_io_memfs__ pharos-cluster
```

Comment:
need to `sudo mkdir` that dir first

Reply:
and the build user needs access

Reply:
> need to sudo mkdir that dir first

Good catch, fixed.

> and the build user needs access

Build user is root.